Cargo.toml

```diff
@@ -65,6 +65,7 @@ futures = "0.3"
 getrandom = { version = "0.2", features = ["js"] }
 hmac = { version = "0.12" }
 http-types = { version = "2.12", default-features = false }
+log = "0.4"
```

Comment:
Do we want to use log or tracing or just standardize on one or the other? Tracing has the advantage that it has an opentelemetry Subscriber, which means you get otel basically for free.

log seems to be more common though.

Reply:
This actually should've have been there. Seems I forgot to `rebase --onto` or something. Cleaning this up...